### PR TITLE
Add cri-dockerd binary to the image

### PR DESCRIPTION
**Pull request by Nino-K**
Signed-off-by: Nino Kodabande <nkodabande@suse.com>

### CODE REVIEW

File: files/build.sh [hunk 1]
```diff
@@ -68,6 +68,17 @@ install rancher-desktop-guestagent /distro/usr/local/bin
...
+# Install cri-dockerd
+mkdir -p /distro/usr/share/doc/cri-dockerd/
+mkdir -p /distro/cri-dockerd/
+tar -xvf /cri-dockerd.tgz -C /distro/cri-dockerd
+chmod u+s /distro/cri-dockerd/cri-dockerd
+cp /distro/cri-dockerd/cri-dockerd /distro/usr/local/bin/
+# Copy the LICENSE file for cri-dockerd
+mkdir -p /distro/usr/share/doc/cri-dockerd/
+cp /distro/cri-dockerd/LICENSE /distro/usr/share/doc/cri-dockerd/
+rm -rf /distro/cri-dockerd
```
Comment (mook-as) [inline files/build.sh:80]: Just extract it _outside_ of `/distro`, and then move the files around:
```suggestion
tar -xvf /cri-dockerd.tgz -C /cri-dockerd
mv /cri-dockerd/cri-dockerd /distro/usr/local/bin/
# Copy the LICENSE file for cri-dockerd
mkdir -p /distro/usr/share/doc/cri-dockerd/
cp /distro/cri-dockerd/LICENSE /distro/usr/share/doc/cri-dockerd/
```
Also, you're creating `/distro/usr/share/doc/cri-dockerd/` twice.

Comment (mook-as) [reply files/build.sh:80]: Now that I think about it, does the executable need to be `u+s`? We're going to run it as root anyway (via OpenRC).

**Review by mook-as — CHANGES_REQUESTED** [review]

File: files/build.sh (hunk 1 above)
Comment (Nino-K) [reply files/build.sh:80]: > Just extract it _outside_ of `/distro`, and then move the files around:

Sure, I will do that, but I still need to create the `/cri-dockerd` directory, the `-C` won't create it correct?

File: files/build.sh [hunk 2]
```diff
@@ -68,6 +68,15 @@ install rancher-desktop-guestagent /distro/usr/local/bin
...
 chroot /distro /sbin/rc-update add rancher-desktop-guestagent default
 
+# Install cri-dockerd
+mkdir /cri-dockerd
+tar -xvf /cri-dockerd.tgz -C /cri-dockerd
+mv /cri-dockerd/cri-dockerd /distro/usr/local/bin/
+# Copy the LICENSE file for cri-dockerd
+mkdir -p /distro/usr/share/doc/cri-dockerd/
+cp /cri-dockerd/LICENSE /distro/usr/share/doc/cri-dockerd/
+rm -rf /cri-dockerd
```
Comment (mook-as) [inline files/build.sh:78]: You don't need to delete this (because all of `/` outside `/distro` is discarded), but that's okay.

**Review by mook-as — APPROVED** [review]

**Pull request merged**